src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlManagedInstanceKeyVaultKey.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/Add-AzureRmSqlManagedInstanceKeyVaultKey
```

Comment:
I'll check these after you have done the parameter sets we agreed upon and regenerated these files.

Reply:
Addressed in my comment below